api/src/database/seeds/03-users.yaml

```diff
@@ -34,7 +34,7 @@ columns:
   language:
     type: string
     length: 8
-    default: 'en-US'
+    default: null
```

Comment:
We shouldn't change seeds (ever). Lets keep this on `en-US`

Reply:
Ok, I'll revert this.